src/main/scala/it/pagopa/interop/backendforfrontend/api/impl/AuthorizationApiServiceImpl.scala

```diff
@@ -79,6 +85,16 @@ final case class AuthorizationApiServiceImpl(
     }
   }
 
+  override def deleteClient(
```

Comment:
it is not wrong to put this here, but this class is focused on actually allow or deny access to the platform, which is a bit sensitive and I would keep it isolated.

May I ask you to move this (and future) authorization process endpoints to a different class? For example to a new class named `ClientsApiServiceImpl`